app/assets/javascripts/collections/story_collection.js

```diff
@@ -18,20 +18,99 @@ module.exports = Backbone.Collection.extend({
     this.labels = [];
   },
 
-  saveSorting: function(columnName) {
+  round: function(value, precision) {
+    var multiplier = Math.pow(10, precision || 0);
+    return Math.round(value * multiplier) / multiplier;
+  },
+
+  getCorrectionFactor: function(this_id, previous_story_id) {
+    var precision = 5;
+    thisStoryPosition = this.round(this.get(this_id).position(), precision);
```

Comment:
It's necessary to use `var` (or better, in this case, `const`) before variables declarations like that, otherwise, you're gonna end up creating a global.